src/index.html

```diff
@@ -17,11 +17,481 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/styles.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!--#region Header-->
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="header__logo-link logo-1200"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
```

Comment:
The `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader). So, it would be better to provide a more descriptive alternative text.